reference/6/About/about_Special_Characters.md

```diff
@@ -112,6 +133,20 @@ The output from this command is:
 
 Column1         Column2         Column3
 
+UNICODE CHARACTER (`u{x})
+The Unicode escape sequence allows you to specify any Unicode character
+including Unicode surrogate characters often used for Emoji e.g. `u{1F44D}.
+The Unicode escape sequence requires at least one hex digit and supports
+up to six hex digits. The maximum hex value is 0x10FFFF.
```

Comment:
Perhaps say "The maximum value is `` `u{10FFFF} ``.", so as to avoid a potential distraction by `0x`?